crates/pathfinder/src/rpc/v02/method/get_class_at.rs

```diff
@@ -62,8 +62,7 @@ pub async fn get_class_at(
 
         let tx = db.transaction().context("Creating database transaction")?;
         let definition = get_definition_at(&tx, block, input.contract_address)?;
-        let class = ContractClass::from_definition_bytes(&definition)
-            .context("Parsing class definition")?;
+        let class = ContractClass::from_definition_bytes(&definition)?;
```

Comment:
Oops, I didn't want to remove `context()` here. Fixing.

Reply:
good catch, didn't notice this :eyes: